js/big-picture.js

```diff
@@ -41,32 +41,37 @@ function renderBigPictureData (pictureData) {
   image.src = pictureData.url;
   likesCount.textContent = pictureData.likes;
   imageDescription.textContent = pictureData.description;
-  commentCountElement.textContent = pictureData.comments.length;
+  commentCountSpan.textContent = pictureData.comments.length;
   createComments(pictureData.comments);
 }
 
 function showBigPicture (pictureData) {
   bigPicture.classList.remove('hidden');
   body.classList.add('modal-open');
-  commentCount.classList.add('hidden');
+  commentCountBlock.classList.add('hidden');
   commentsLoader.classList.add('hidden');
   renderBigPictureData(pictureData);
-}
 
-function hideBigPicture () {
-  bigPicture.classList.add('hidden');
-  body.classList.remove('modal-open');
+  document.addEventListener('keydown', onDocumentKeydown);
 }
 
-closeButton.addEventListener('click', function() {
-  hideBigPicture();
-});
-
-document.addEventListener('keydown', function(evt) {
+function onDocumentKeydown (evt) {
   if (isEscEvent(evt)) {
     evt.preventDefault();
-    hideBigPicture();
+    hideModal(bigPicture);
   }
-});
+}
+
+function hideModal (modal) {
```

Comment:
```suggestion
function hideBigPicture () {
```

это функция - антагонист для функции `showBigPicture`. Они идут в связке и из названия это должно быть видно.

Обрати внимание, что при показе большой картинки у тебя 5 действий (рендер не считаем), а при скрытии только 3 действия - куда-то делось еще 2 действия